### DISCUSSION TITLE
Writing Mode and Text Orientation classes

### DISCUSSION THREAD

**Discussion by yp717** [tailwindlabs/tailwindcss#5237] [Ideas]
Hi!

I've recently come across a use-case where the writing mode and text-orientation CSS properties were very useful and they are not directly available in Tailwind. I think a new set of utility classes for this could make a nice addition for the future.

https://developer.mozilla.org/en-US/docs/Web/CSS/text-orientation
https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode

Happy to have a go at implementing it and making the PR but I wanted to see whether this was anyone else was interested in too! :)

**Comment by mrienstra** [comment]
Related discussions:
https://github.com/tailwindlabs/tailwindcss/discussions/2494
https://github.com/tailwindlabs/tailwindcss/discussions/6845
https://github.com/tailwindlabs/tailwindcss/discussions/9240
